### ISSUE TITLE
.gitignore equivalent?

### ISSUE THREAD

**Issue by ebenolson** [robmadole/jig#12]
The whitespace plugin seems to hang on large .ipynb files, it would be nice to have a .jigignore to skip processing of certain files.

**Comment by robmadole** [comment]
@ebenolson I agree. I've thought of this before but didn't prioritize it very high. I'll move it up in the list. Thanks.
